hybrid_assignment.py

```diff
@@ -98,7 +98,7 @@ def list_projects_for_user(self, user_id, group_ids, hints):
         # Make sure the default project is in the project list for the user
         # user_id
         for project in projects:
-            if project.id == self.default_project_id:
+            if project.get('id') == self.default_project_id:
```

Comment:
This was just fixed yesterday. See: https://github.com/SUSE-Cloud/keystone-hybrid-backend/pull/22